src/web/next.config.js

```diff
@@ -50,3 +52,10 @@ module.exports = withMDX({
   reactStrictMode: true,
   trailingSlash: true,
 });
+
+module.exports = withPWA({
+  pwa: {
```

Comment:
This is why the about page wasn't rendering (noted in my screenshot comment), I would love to see the solution to this

Reply:
@chrispinkney : I am working on it